src/skins/vector/css/themes/_base.scss

```diff
@@ -104,6 +104,8 @@ $roomtile-name-color: rgba(69, 69, 69, 0.8);
 $roomtile-selected-bg-color: rgba(255, 255, 255, 0.8);
 $roomtile-focused-bg-color: rgba(255, 255, 255, 0.9);
 
+$roomtile-transparent-focused-color: rgba(0, 0, 0, 0.1);
```

Comment:
why are we darkening it for focus? the rest of the UI brightens things for focus...

Reply:
In the light theme it has a white background to begin with, so we can't go any brighter. Would you propose something else?

Reply:
well, we could lighten the foreground even if the background is already saturated. i'll merge it as is though and see how it looks and tweak as needed then.